pkg/bundle/verification_content.go

```diff
@@ -23,8 +23,8 @@ import (
 	"github.com/sigstore/sigstore-go/pkg/verify"
 )
 
-type CertificateChain struct {
-	Certificates []*x509.Certificate
+type Certificate struct {
```

Comment:
Again I think we need both `CertificateChain` and `Certificate`.

Reply:
Ah, I just noticed we don't actually support providing intermediates, we assume the chain contains a single certificate - https://github.com/sigstore/sigstore-go/blob/28ec3ea19b7b93706aa8403398fc40c6394bb4d6/pkg/bundle/verification_content.go#L51-L53

So then +1 to this struct change, and can we file an issue tracking adding support for intermediates?

Reply:
Sure! https://github.com/sigstore/sigstore-go/issues/132